src/content/migrate/5.md

```diff
@@ -0,0 +1,152 @@
+
+
+## Preparations
+
+webpack 5 requires at least Node.js 10.13.0 (LTS).
+
+T> Using newer node.js version can improve build performance.
+
+### Upgrade webpack and its dependencies
+
+#### Upgrade webpack 4 to the latest available version
+
+When using webpack >= 4, upgrading to the latest webpack 4 version should not require additional guidance.
+If you are using webpack version less than 4 please see the [webpack 4 migration guide](/migrate/4/).
+
+#### Upgrade webpack-cli to the latest available version (when used)
+
+#### Upgrade all used Plugins and Loaders to the latest available version
+
+Some Plugins and Loaders might have a beta version that has to be used in order to be compatible with webpack 5.
+
+W> Check related Plugins and Loaders migration guide when upgrading across major versions.
+
+#### Make sure your build has no errors or warnings
+
+There might be new errors or warnings because of the upgraded versions of webpack, webpack-cli, Plugins and Loaders. Keep an eye for deprecation warnings during the build.
+
+You can invoke webpack this way to get stack traces for deprecation warnings to figure out which Plugins and Loaders are responsible.
+
+```bash
+node --trace-deprecation node_modules/webpack/bin/webpack.js
+```
+
+W> webpack 5 removes all deprecated features. In order to proceed you must have no webpack deprecation warnings during the build.
+
+#### Make sure you are using entry point information from stats
+
+T> If you are using [HtmlWebpackPlugin](/plugins/html-webpack-plugin/) skip this step.
+
+When using static HTML or creating HTML in other way, make sure to use entry points from stats JSON to generate <script>, <style> and <link> tags.
+
+If this is not possible, avoid setting `splitChunks.chunks: 'all'` and `splitChunks.maxSize` later in this guide. Note that this is sub-optimal and a workaround.
+
+#### Make sure to use `mode`
+
+Set mode to either [`production`](/configuration/mode/#mode-production) or [`development`](/configuration/mode/#mode-development) to make sure that corresponding defaults set.
+
+#### Update outdated options
+
+Update the following options to their new version (if used):
+
+- `optimization.hashedModuleIds: true` => `optimization.moduleIds: 'hashed'`
+- `optimization.namedChunks: true` => `optimization.chunkIds: 'named'`
+- `optimization.namedModules: true` => `optimization.moduleIds: 'named'`
+- `NamedModulesPlugin` => `optimization.moduleIds: 'named'`
+- `NamedChunksPlugin` => `optimization.chunkIds: 'named'`
+- `HashedModulesPlugin` => `optimization.moduleIds: 'hashed'`
+- `optimization.occurrenceOrder: true` => `optimization: { chunkIds: 'total-size', moduleIds: 'size' }`
+
+#### Disable ES2015 syntax in runtime code, if necessary
+
+By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build target's environments that don't support this syntax (like IE11), you'll need to set `output.ecmaVersion: 5` to revert to ES5 syntax.
```

Comment:
This step is too early here. webpack 4 doesn't support `output.ecmaVersion`, so this should only be done after upgrading to webpack 5